style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.menu {
+  background-color: #3c3a39;
+  background-image: url(./image/toggel.svg);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  color: white;
+  padding: 12px;
+  margin: 24px;
+}
+
+nav {
+  background-color: #3c3a39;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 92px;
+  position: absolute;
+  width: 375px;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav.logo {
+  margin-bottom: 2px;
+  margin-left: 12px;
+}
+
+.header-container {
+  background: #1c1a19;
+  background-image: url("./image/Illustration.svg");
+  background-position-x: right;
+  background-position-y: bottom;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 120px 24px 120px 24px;
+}
+
+h1,
+h2 {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  display: flex;
```

Comment:
- Please make sure that your project sticks as much as possible to the Figma template provided. Kindly reduce the space between the lines of the title. :+1: :smiley:  :smiley: 
- Also make sure please to use the font provided by the Figma file. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192105165-a6978062-c03a-43e6-9ad6-299fa6a5f7a3.png)